### PR TITLE
Addition of span count attribute

### PR DESCRIPTION
**Pull request by sarthak77**
Ref issue : https://github.com/hypertrace/hypertrace-ui/issues/1157

### CODE REVIEW

File: hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl [hunk 1]
```diff
@@ -70,6 +70,9 @@ protocol SpanEventViewProtocol {
     // count of api_trace
     int api_trace_count = 0;
 
+    // count of span
+    int span_count = 0;
```
Comment (kotharironak) [inline hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl:74]: Add at the end, after last attribute.

Comment (sarthak77) [reply hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl]: resolved

File: hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl [hunk 2]
```diff
@@ -103,5 +103,8 @@ protocol SpanEventViewProtocol {
     map<string> api_callee_name_count = {};
 
     int api_trace_error_span_count = 0;
+
+    // count of span
```
Comment (kotharironak) [inline hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl:107]:
```suggestion
```

File: hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java [hunk 3]
```diff
@@ -226,6 +226,9 @@ private Event getApiEntrySpanForExitSpan(
       builder.setErrorCount((int) errorMetric.getValue().doubleValue());
     }
 
+    // span count
```
Comment (kotharironak) [inline hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java:229]:
```suggestion
```

**Review by kotharironak — APPROVED** [review]

File: hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl [hunk 4]
```diff
@@ -103,5 +103,7 @@ protocol SpanEventViewProtocol {
     map<string> api_callee_name_count = {};
 
     int api_trace_error_span_count = 0;
+
+    int span_count = 0;
```
Comment (kotharironak) [inline hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl:107]: `span_count` -> `spans` ?

Comment (sarthak77) [reply hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl]: done

**Review by kotharironak — APPROVED** [review]

**Pull request merged**